### PR TITLE
Tao 797 authoring form

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

File: helpers/Apip.php [hunk 1]
```diff
@@ -62,9 +84,13 @@ static public function extractApipAccessibility(DOMDocument $doc)
...
-    {
-        $newNode = $qtiItem->importNode($apipContent->documentElement, true);
-        $qtiItem->documentElement->appendChild($newNode);
+    {        
+        $apipXMLString = $apipContent->saveXML($apipContent->documentElement);
+        
+        $fragment = $qtiItem->createDocumentFragment();
+        $fragment->appendXml($apipXMLString);
+        
+        $qtiItem->documentElement->appendChild($fragment);
```
Comment (ssipasseuth) [inline helpers/Apip.php]: what is the difference between this version (saveXML+createDocumentFragment+appendXml) and the previous one (importNode+appendChild)

Comment (hutnikau) [inline helpers/Apip.php]: Appending apip content using previous approach removes prefixes (`apip:*`) from apip content elements.

Comment (ssipasseuth) [inline helpers/Apip.php]: ok, thanks for the clarification

**Comment by ssipasseuth** [comment]
There are some missing unit tests (for the forms) + styling issues.
That would be done as soon as the libs are proven stable enough.

**Pull request merged**